packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 3.2.0
```

Comment:
According to our new separate release handling of dart / flutter, changes to dart should not appear in the flutter changelog anymore. The only changelog entry for flutter I guess is that some dependencies were upgraded?

Reply:
Modifications were made to both packages, so the versions are the same.

My opinion. In a next release, it would match the versions one more time (4.0.0?), then each package would have its own versioning.

But it's hard not to stay in sync.

Both packages always have dependencies that must be updated frequently and most of the bugs in the Flutter layer are fixed in the Dart package.